scripts/release-post.js

```diff
@@ -71,41 +72,56 @@ function findLatestVersion (cb) {
 
 function fetchDocs (version) {
   return Promise.all([
-    fetchChangelog(version),
+    fetchChangelogBody(version),
+    fetchAuthor(version),
     fetchShasums(version),
     verifyDownloads(version)
   ]).then(function (results) {
     const changelog = results[0]
-    const shasums = results[1]
-    const files = results[2]
+    const author = results[1]
+    const shasums = results[2]
+    const files = results[3]
 
     return {
       version,
       changelog,
+      author,
       shasums,
       files
     }
   })
 }
 
+function fetchAuthor (version) {
+  return fetchChangelog(version)
+    .then((section) => findAuthorLogin(version, section))
+    .then((author) => download(`https://api.github.com/users/${author}`))
+    .then(JSON.parse)
+    .then((githubRes) => githubRes.name)
+}
+
 function fetchChangelog (version) {
+  // matches a complete release section,
+  // support release sections with headers like:
+  // ## 2015-09-22, Version 4.1.1 (Stable), @rvagg
+  const rxSection = new RegExp(`## \\d{4}-\\d{2}-\\d{2}, Version ${version} \\(Stable\\)[\\s\\S]*?(?=## \\d{4})`)
```

Comment:
We're going to need to extend this, it'll include "LTS" and "Maintenance" too. Perhaps just `\\([^\\)]+\\)` would do.